packages/flutter_tools/gradle/src/main/groovy/flutter.groovy

```diff
@@ -458,31 +458,32 @@ class FlutterPlugin implements Plugin<Project> {
     private Boolean doesSupportAndroidPlatform(String path) {
         File buildGradle = new File(path, 'android' + File.separator + 'build.gradle')
         File buildGradleKts = new File(path, 'android' + File.separator + 'build.gradle.kts')
-        if (settingsGradle.exists() && settingsGradleKts.exists()) {
+        if (buildGradle.exists() && buildGradleKts.exists()) {
             logger.error(
-                "Both build.gradle and build.gradle.kts exist. " +
-                "This is not supported and will result in an error in the future."
+                "Both build.gradle and build.gradle.kts exist, so the " +
+                "latter one is ignored. This is likely a mistake."
```

Comment:
Nit: I think we should be specific here and elsewhere. "build.gradle.kts is ignored. This ...".

Reply:
Fixed in 4e32f79482a7aa3ebd1a154bb6529e5c6282bea5